### DISCUSSION TITLE
core package for ie support

### DISCUSSION THREAD

**Discussion by vaibhavkhadake** [vercel/next.js#13790] [Help]
I install core package and i imported in _document file but won't use .
whats going wrong i don't understand

**Comment by jamesmosier** [comment]
Could you provide a repository showing what you are having issues with? 

Also, I assume you are talking about `core-js`? If so, Next already contains [a ton of polyfills](https://github.com/vercel/next.js/blob/88fd6cf31d919a60ad9c25604812b06802c47511/packages/next-polyfill-nomodule/src/index.js) for IE11. Are there any in particular you want that are not offered?

Lastly, if you really want to import *more* polyfills for **all browsers** then you'd want to look at [this polyfills example](https://github.com/vercel/next.js/blob/canary/examples/with-polyfills/README.md)

**Reply by vaibhavkhadake** [reply]
![navlist](https://user-images.githubusercontent.com/53461582/83935920-c6034180-a7db-11ea-9c28-b6fae2d875a8.png)
YES i am talking about core js.
When i click on direct button NAV value change which is stored in redux store.
In chrome and firefox its working but ie not working.
thanks for replying

**Reply by rafaelalmeidatk** [reply]
Which polyfill do you want do include? This doesn't seem to be an error with polyfills tbh

**Reply by vaibhavkhadake** [reply]
plz refer :https://www.npmjs.com/package/core-js  this polyfill i want to use.
because chrome it work absolutely fine problem in IE only.so i didn't understand
when we get these type of error any idea?
